pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -855,10 +865,22 @@ public String getDescription() {
    * @param properties properties to take actual value from
    * @return evaluated value for this connection parameter
    */
+  // todo should be renamed to getOrDefault()
   public @Nullable String get(Properties properties) {
     return properties.getProperty(name, defaultValue);
   }
 
+  /**
+   * Returns the value of the connection parameters according to the given {@code Properties}
+   *
+   * @param properties properties to take actual value from
+   * @return evaluated value for this connection parameter
+   */
+  // todo should be renamed to get()
```

Comment:
why isn't it renamed to get ?

Reply:
before that `get()` has to be renamed to `getOrDefault()`.
As result 18 files will be changed.
I did not want to do it without wider approval (maybe you have different ideas/concepts)
I case you approve those 2 renames then I will do it.